cocos/platform/CCFileUtils.cpp

```diff
@@ -860,9 +860,11 @@ std::string FileUtils::fullPathFromRelativeFile(const std::string &filename, con
 void FileUtils::setSearchResolutionsOrder(const std::vector<std::string>& searchResolutionsOrder)
 {
     bool existDefault = false;
+    std::vector<std::string> searchResolutionOrderCopied = searchResolutionsOrder;
+
     _fullPathCache.clear();
     _searchResolutionsOrderArray.clear();
-    for(const auto& iter : searchResolutionsOrder)
+    for(const auto& iter : searchResolutionOrderCopied)
```

Comment:
For setSearchResolutionsOrders, we could.
But for setSearchPaths, we couldn't.
Because setSearchPaths will update `_searchPathArray` which will be affected by `_defaultResourceRootPath`.